src/OpenTelemetry.Instrumentation.Runtime/README.md

```diff
@@ -90,7 +90,7 @@ The value is available even before first garbage collection has occurred.
 
 The API used to retrieve the value is:
 
-* [GC.GetTotalMemory](https://docs.microsoft.com/dotnet/api/system.gc.gettotalmemory):
+* [`GC.GetTotalMemory`](https://docs.microsoft.com/dotnet/api/system.gc.gettotalmemory):
```

Comment:
These don't seem to be violating MD034 - Bare URL used https://github.com/updownpress/markdown-lint/blob/master/rules/034-no-bare-urls.md

That aside, I don't have opinion against surrounding with the backticks. But if you do, would you add to all the APIs in this doc?

Reply:
I have found another way to fix the issue without reformatting this file.

See: 8062bb6e1ecc1c94b1ad7c9bde3ef2b0d86b667a

Ref. link to issues external GH issue: https://github.com/DavidAnson/markdownlint/issues/787